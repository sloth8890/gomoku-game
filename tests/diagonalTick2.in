place A5
place S1
place B4
place R3
place C3
place R1
place D2
place C1
place E1