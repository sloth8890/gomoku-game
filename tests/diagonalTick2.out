Black wins!
A5S1B4R3C3R1D2C1E1
Thank you for playing!
